tor_ocr/strings.py

```diff
@@ -6,3 +6,8 @@
     'override whatever I provide, so please format my content accordingly '
     'if you choose to use it. '
 )
+
+reply_response = (
+    'Hello, I do not respond to commands. Perhaps you meant to reply to '
+    'u/transcribersofreddit instead.'
```

Comment:
To make sure that this doesn't accidentally ping the other bot, we should probably escape the username with a backslash:
`r'u\/transcribersofreddit instead.'`